index.html

```diff
@@ -3117,25 +3122,22 @@ <h4><code>unsubscribeevent</code></h4>
             the Web Thing with:
           </p>
           <ul>
-            <li>Method set to <code>POST</code></li>
-            <li>URL set to the URL of the <code>Event</code> resource</li>
+            <li>Method set to <code>DELETE</code></li>
+            <li>URL set to the URL of the <code>Event</code> resource followed by the <code>subscriptionId</code> of a valid subscription</li>
```

Comment:
[Example 44](https://w3c.github.io/wot-profile/#example-44) uses `subscriptionID`, but this says `subscriptionId`.

Reply:
What should the response be? I would suggest `204 No Content`, as with the cancelaction protocol binding in the HTTP Basic Profile. This would also mean no `Accept` header is needed on this request.